src/main/resources/anniv5Resources/localization/eng/serpentinepack/Powerstrings.json

```diff
@@ -14,8 +14,8 @@
     "NAME": "Tainted",
     "DESCRIPTIONS": [
       "Doubles any debuffs you apply to this enemy.",
-      "Doubles any debuffs you apply to this enemy #b",
```

Comment:
The old description wasn't even accurate -- it multiplied by 2x, but the way to double something _x_ times is to multiply by 2<sup>x</sup>. This is probably why the ZHS string was set up differently, to give a more accurate description.